Jenkinsfile

```diff
@@ -397,112 +397,112 @@ try {
           parallel_containers["${windows_containers[i].os}-${windows_containers[i].flavor}"] = {
             def current_container = windows_containers[index]
             node('windows') {
-              stage('prepare container') {
-                checkout scm
-                docker.withRegistry('https://263245908434.dkr.ecr.us-east-1.amazonaws.com', 'ecr:us-east-1:jenkins-aws') {
-                  def image_tag = "${current_container.os}-${rstudioReleaseBranch}"
-                  windows_image = docker.image("jenkins/ide:" + image_tag)
-
-                  retry(5) {
-                    windows_image.pull()
-                    image_name = windows_image.imageName()
-                  }
-                }
-              }
-              docker.image(image_name).inside() {
-                stage('dependencies') {
-                    withCredentials([usernameColonPassword(credentialsId: 'github-rstudio-jenkins', variable: "GITHUB_LOGIN")]) {
-                      bat 'cd dependencies/windows && set RSTUDIO_GITHUB_LOGIN=$GITHUB_LOGIN && set RSTUDIO_SKIP_QT=1 && install-dependencies.cmd && cd ../..'
+
+              retry(5) {
```

Comment:
This doesn't help if the agent goes away. I think you want to wrap `node{'windows') {...}`, and start off with a value of just 2.